notebooks/aws-emr/init-configurations.json

```diff
@@ -0,0 +1,87 @@
+[
+{
+	"Classification":"spark",
+		"Properties":{
+			"enableSparkRapids":"true"
+		}
+},
+{
+	"Classification":"yarn-site",
+	"Properties":{
+		"yarn.nodemanager.resource-plugins":"yarn.io/gpu",
+		"yarn.resource-types":"yarn.io/gpu",
+		"yarn.nodemanager.resource-plugins.gpu.allowed-gpu-devices":"auto",
+		"yarn.nodemanager.resource-plugins.gpu.path-to-discovery-executables":"/usr/bin",
+		"yarn.nodemanager.linux-container-executor.cgroups.mount":"true",
+		"yarn.nodemanager.linux-container-executor.cgroups.mount-path":"/sys/fs/cgroup",
+		"yarn.nodemanager.linux-container-executor.cgroups.hierarchy":"yarn",
+		"yarn.nodemanager.container-executor.class":"org.apache.hadoop.yarn.server.nodemanager.LinuxContainerExecutor"
+	}
+},
+{
+	"Classification":"container-executor",
+	"Properties":{
+
+	},
+	"Configurations":[
+	{
+		"Classification":"gpu",
+		"Properties":{
+			"module.enabled":"true"
+		}
+	},
+	{
+		"Classification":"cgroups",
+		"Properties":{
+			"root":"/sys/fs/cgroup",
+			"yarn-hierarchy":"yarn"
+		}
+	}
+	]
+},
+{
+	"Classification":"spark-defaults",
+	"Properties":{
+		"spark.plugins":"com.nvidia.spark.SQLPlugin",
+		"spark.sql.sources.useV1SourceList":"",
+		"spark.executor.resource.gpu.discoveryScript":"/usr/lib/spark/scripts/gpu/getGpusResources.sh",
```

Comment:
Can you check formatting on this file?  (tabs vs spaces, etc)?  Looks like entries are misaligned.

Reply:
Replaced all tab with four space using auto alignment vim command 'gg=G'.